src/app/attendance/actions/attendanceActions.tsx

```diff
@@ -0,0 +1,26 @@
+"use server";
+import { AttendanceStatus, Table } from "@/types/index";
+import { getSupabaseClient } from "../../../../utils/supabase/supabaseClient";
+import { redirect } from "next/navigation";
+import { ATTENDANCE_PATH } from "@/constant/constant";
```

Comment:
This should be coming from routes. We've enum defined there.

Reply:
Fix